SimPEG/PF/Magnetics.py

```diff
@@ -39,11 +39,11 @@ def fwr_ind(self, m):
 
         else:
 
-            return self.G.dot(m)
+            return self.G.dot(self.chiMap * m)
```

Comment:
here, you can replace `self.chiMap * m` with `self.chi` (does the mapping step for you)

Reply:
same elsewhere